etc/docker-scripts/bootstrapper.py

```diff
@@ -324,13 +341,24 @@ def parse_arguments(cls, args) -> dict:
                             help='Directory in Volume to install python libraries into', required=False)
         parsed_args = vars(parser.parse_args(args))
 
+        pipeline_name = parsed_args.get('cos-directory')  # cos-directory is the pipeline name, set as global
         return parsed_args
 
+    @classmethod
+    def log_operation_info(cls, action_clause: str, duration_secs: Optional[float] = None):
+        global pipeline_name
+        duration_clause = f"({duration_secs:.3f} secs)" if duration_secs else ""
+        logger.info(f"kfp pipeline '{pipeline_name}' - {action_clause} {duration_clause}")
```

Comment:
This is always in the context of the same pipeline and node, so printing this again and again on each line is overkilling and use important real estate for the actual debug messages we are trying to print.

Reply:
> use important real estate for the actual debug messages we are trying to print

Please understand that these "formatted" entries are specific informational messages.  They are not to be meant to be troubleshooting/debugging log statements (although they do provide useful information).  Other logging can take place directly without the preamble - although all log entries will have the date/timestamp and application name - which is really a majority of the real estate.

I agree there's some redundancy between _these_ log statements. I was trying to address three things:  
1. A consistent look relative to the logs from the jupyter server.  
2. An assumption that logs are aggregated, including those from the jupyter server so support scripts could be developed to grep specific "high-level" strings (namely pipeline name) and get all relative entries. 
3. When support folks ask for "logs" they will get several and can't rely on the customer organizing logs to reflect the appropriate pipelines and/or nodes/operations because that is rare for a customer to do.  (I have literally experienced years of that.)

Item 1 isn't a big deal **provided that the same search string can be used across all sets of logs** to identify the same pipeline.  Items 2 and 3, I believe, are important.

I agree that these consume valuable real estate and I am okay trimming them down.  I do think though that log aggregation will be a common thing across adopting applications and that we should make that easy. For that reason, I believe we should keep the pipeline name **and** convey the operation/node name to kfp-notebook for the sole purpose of including it in these logs.  

I can also remove/change the application name - thinking of changing to 'elyra' so the new entries would resemble something like these.  (Note I've kept the pipeline name and added the node name using dotted notation.)
```
[I 2020-10-02 22:48:34,936.936 elyra] p2-1002154733.node1 - processing outputs 
[I 2020-10-02 22:48:34,943.943 elyra] p2-1002154733.node1 - uploaded node1a.out to bucket: kbates object: p2-1002154733/node1a.out (0.007 secs)
[I 2020-10-02 22:48:34,949.949 elyra] p2-1002154733.node1 - uploaded node1b.out to bucket: kbates object: p2-1002154733/node1b.out (0.006 secs)
```
Please know that I agree there's real estate consumed here, but I also believe there are good reasons to have these entries that may not be evident in our development-mode interactions.  I will submit a commit to apply some "trimming".

If you disagree that the use cases I point out are not valid or should be addressed in another manner, I happy to have that discussion.

I'd like to hear @ptitzler's opinions since he opened this issue in the first place.